contrib/terraform/upcloud/main.tf

```diff
@@ -38,6 +38,12 @@ module "kubernetes" {
   loadbalancer_outbound_proxy_protocol = var.loadbalancer_proxy_protocol ? "v2" : ""
   loadbalancers                        = var.loadbalancers
 
+  connect_router   = var.connect_router
+  gateways         = var.gateways
+  gateway_vpn_psks = var.gateway_vpn_psks
+  static_routes    = var.static_routes
+  network_peerings = var.network_peerings
+
```

Comment:
Were you planning on adding some default values to cluster-settings.tfvars for these variables ?

Reply:
No, they are have default values that won't change the current behaviour so I thought that wasn't needed. But if you want I can add it

Reply:
Looks like the values you _would_ set in `cluster-settings.tfvars` would just be `{}` right? In that case it could be skipped

On the other hand we do that for other values e.g. [server_groups](https://github.com/elastisys/kubespray/blob/d99116974f8f54d5697c49559900d5f51bf15434/contrib/terraform/upcloud/cluster-settings.tfvars#L140)